src/pages/scientificServices/pipelines/components/inputs/PipelineStringInput.tsx

```diff
@@ -18,12 +18,17 @@ export const PipelineStringInput: React.FC<PipelineStringInputProps> = ({
   validationError,
   onValidation,
 }) => {
-  const { label, placeholder, helpText, validationRegex } = INPUT_DESCRIPTIONS[input.name] || {};
+  const { label, placeholder, helpText, validationRegex } = INPUT_DESCRIPTIONS[input.name];
 
   return (
     <>
       <h3 style={{ marginBottom: '0.5rem' }}>
-        {label || input.name} {input.isRequired ? <span style={{ color: '#DB3214' }}>*</span> : null}
+        {label || input.name}
+        {input.isRequired ? (
+          <span style={{ color: '#DB3214' }}>*</span>
```

Comment:
nit: this lost a separating space


```suggestion
          <span style={{ color: '#DB3214' }}> *</span>
```